core/spec/models/spree/user_spec.rb

```diff
@@ -92,7 +92,9 @@
       end
 
       it "has payment sources" do
-        expect(user.payment_sources.first.gateway_customer_profile_id).not_to be_empty
+        ActiveSupport::Deprecation.silence do
```

Comment:
We don't want to silence `ActiveSupport::Deprecation` but `Spree::Deprecation` here